x-pack/plugins/security_solution/common/endpoint/models/policy_config.ts

```diff
@@ -12,6 +12,14 @@ import { PolicyConfig, ProtectionModes } from '../types';
 export const factory = (): PolicyConfig => {
   return {
     windows: {
+      advanced: {
```

Comment:
I'm thinking that we remove these defaults from the Policy config and treat advanced as a special case.  @ferullo would prefer that we don't actually send any value for advanced stuff unless the user explicitly sets it.

Reply:
Agreed. I think we shouldn't send advanced options from Kibana to Endpoint unless its explicitly set in policy.

Reply:
23238e9d29504f34a9e774cad99443b830951c52